<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditor</class>
 <widget class="QWidget" name="ScriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_nameLabel">
     <property name="text">
      <string>Script name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="m_nameEdit"/>
   </item>
   <item row="1" column="0" rowspan="8" colspan="3">
    <widget class="QTextEdit" name="m_editor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="tabStopWidth">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="m_addButton">
     <property name="toolTip">
      <string>Add new command to cursor position</string>
     </property>
     <property name="icon">
      <iconset resource="qlcui.qrc">
       <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="m_cutButton">
     <property name="toolTip">
      <string>Cut selected text to clipboard</string>
     </property>
     <property name="icon">
      <iconset resource="qlcui.qrc">
       <normaloff>:/editcut.png</normaloff>:/editcut.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="m_copyButton">
     <property name="toolTip">
      <string>Copy selected text to clipboard</string>
     </property>
     <property name="icon">
      <iconset resource="qlcui.qrc">
       <normaloff>:/editcopy.png</normaloff>:/editcopy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QToolButton" name="m_pasteButton">
     <property name="toolTip">
      <string>Paste text from clipboard at cursor</string>
     </property>
     <property name="icon">
      <iconset resource="qlcui.qrc">
       <normaloff>:/editpaste.png</normaloff>:/editpaste.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QToolButton" name="m_undoButton">
     <property name="toolTip">
      <string>Undo</string>
     </property>
     <property name="icon">
      <iconset resource="qlcui.qrc">
       <normaloff>:/undo.png</normaloff>:/undo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>29</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qlcui.qrc"/>
  <include location="qlcui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_cutButton</sender>
   <signal>clicked()</signal>
   <receiver>m_editor</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyButton</sender>
   <signal>clicked()</signal>
   <receiver>m_editor</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pasteButton</sender>
   <signal>clicked()</signal>
   <receiver>m_editor</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_undoButton</sender>
   <signal>clicked()</signal>
   <receiver>m_editor</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
